bridge/framework/django.py

```diff
@@ -62,16 +64,52 @@ def configure_staticfiles(self, platform: Platform):
                     "whitenoise.storage.CompressedManifestStaticFilesStorage"
                 )
 
+    def configure_worker(self, platform: Platform) -> None:
+        environment = build_redis_environment(platform)
+        self.framework_locals["CELERY_BROKER_URL"] = environment.url()
+        self.framework_locals["CELERY_RESULT_BACKEND"] = environment.url()
+
+    def start_local_worker(self) -> None:
+        # Confirm we are in a `runserver` command
+        if sys.argv[1] == "runserver":
```

Comment:
The django management util was printing out a bunch of stuff and wasn't easy to silence, so have a basic check here for now